### PR TITLE
chore: data seeding for multiple schools

### PR DESCRIPTION
**Pull request by Kelv1nG**
🎫 Ticket

chore: data seeding for multiple schools

📦 Packages

None

📝 Updates

add seeders to simulate multiple schools / teachers / students

**School**
count = 2
```
for each school:
    3 years with 2 sems ( 6 academic years )

    10 grade levels
        2 section per grade level
        6 subjects per grade level

        6 subjects allocated per section
        20 students per section
```

**Teacher**
count = 20 / school

above calc is based on:
    6 subjects * 10 grade level * 2 section  / ( 3 subjects * 2 section per teacher) = 120 / 6 = 20

```    
for each teacher:
    3 subjects assigned to 2 sections

    1 assessment per assessment type and assessment status
    2-3 questions per assessment         
```

**Student**
   count = 2400 / school
   
   above calc is  based on:
         10 grade levels * 2 sections * 20 students per section = 400
         400 fresh students * 3 years * 2 semester = 2400
```         
for each student:
   6 allocated subjects
   1 section assigned
        
    some  students dont have assessments submitted
    some  students have incomplete answered questions to assessments
   ```
🧪 Testing Steps
       N/A

### CODE REVIEW

File: src/app/Enums/DayOfWeek.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
```
Comment (ejempic) [inline src/app/Enums/DayOfWeek.php:1]: We could use carbon but its okay

File: src/database/seeders/AssessmentQuestionAnswerKeySeeder.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            for ($i = 0; $i < 3; $i++) {
+                $answerRecords[] = AssessmentQuestionAnswer::factory()->make([
+                    'question_id' => $multipleChoiceQuestion->id,
+                    'is_correct' => false,
+                ])->attributesToArray();
+            }
+        }
+
+        // prepare true or false
+        $trueOrFalseQuestions = AssessmentQuestion::where('question_type', 'true_or_false')->get();
```
Comment (ejempic) [inline src/database/seeders/AssessmentQuestionAnswerKeySeeder.php:48]: All are true

File: src/database/seeders/AssessmentQuestionAnswerKeySeeder.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            ])->attributesToArray();
+        }
+
+        // prepare short answer
+        $shortAnswerQuestions = AssessmentQuestion::where('question_type', 'short_answer')->get();
+        foreach ($shortAnswerQuestions as $shortAnswerQuestion) {
+            for ($i = 0; $i < 3; $i++) {
+                $answerRecords[] = AssessmentQuestionAnswer::factory()->make([
+                    'question_id' => $shortAnswerQuestion->id,
+                    'is_correct' => true,
```
Comment (ejempic) [inline src/database/seeders/AssessmentQuestionAnswerKeySeeder.php:62]: All are true

**Review by ejempic — CHANGES_REQUESTED** [review]

File: src/database/seeders/AssessmentQuestionAnswerKeySeeder.php (hunk 2 above)
Comment (Kelv1nG) [reply src/database/seeders/AssessmentQuestionAnswerKeySeeder.php:48]: answer field updated to be true or false

for true or false questions, i think only the correct answer should be saved in the answer key, as the incorrect answer is implied.

update: [7beab](https://github.com/Kelv1nG/iskulapp-laravel/pull/26/commits/7beab80808e1fb73c57458d4a8fbed90b1fc46c7)

File: src/database/seeders/AssessmentQuestionAnswerKeySeeder.php (hunk 3 above)
Comment (Kelv1nG) [reply src/database/seeders/AssessmentQuestionAnswerKeySeeder.php:63]: this is intended, as this corresponds to 3 acceptable answers based on student input

i.e 
question: 
    who is the current president of our country?


accepted answers:

short answer1: BBM!
short answer2: BBM
short answer3: Mahal na Presidenteng BongBong Marcos Jr.

File: src/app/Enums/DayOfWeek.php (hunk 1 above)
Comment (Kelv1nG) [reply src/app/Enums/DayOfWeek.php]: enum removed

**Pull request merged**